front/app/src/js/utils/JSONRequests.js

```diff
@@ -0,0 +1,31 @@
+
+export class JSONRequests {
+  static async get(url, headers={}) {
+    const options = {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+        ...headers,
+      },
+    };
+    return await JSONRequests.request(url, options);
+  }
+
+  static async post(url, body, headers={}) {
+    const options = {
+      method: 'POST',
+      body: JSON.stringify(body),
+      headers: {
+        'Content-Type': 'application/json',
+        ...headers,
+      },
+    };
+    return await JSONRequests.request(`${url}`, options);
```

Comment:
Those two lines are formatted differently even though they seem to do the same thing.
`return await JSONRequests.request(url, options);`
`return await JSONRequests.request(`${url}`, options);`

Reply:
https://github.com/tdameros/42-transcendence/pull/122/commits/e55890be33c820452fe593ac85a51b42df768be5